### PR TITLE
fix Dropbox adapter

### PR DESCRIPTION
**Pull request by mikk150**

### CODE REVIEW

**Comment by schmunk42** [comment]
I've never used `DropboxAdapter` ... did you test these changes with an actual Dropbox-Account?

**Comment by mikk150** [comment]
> I've never used `DropboxAdapter` ... did you test these changes with an actual Dropbox-Account?

well, it does not throw an exception.. but I did not get it to work. Dropbox just did not allow me to upload.. I have no idea why

**Comment by schmunk42** [comment]
Has the Dropbox library changed this recently?
I wonder if we'd need to require a minimum version of the lib.

**Comment by mikk150** [comment]
ok, disregard, it does indeed work... I am just doing something stupid here with my tokens. If I took dropbox token from app that has working dropbox connection, it works.

**Comment by schmunk42** [comment]
So ready to merge from your side?

**Comment by mikk150** [comment]
> So ready to merge from your side?

I'm trying to see, if I can maybe use yii2-authclient to get token that actually works

**Comment by mikk150** [comment]
and it now works with new token as well.. It was bad UX on Dropbox side.. Cookie policy was blocking submit button....

**Comment by mikk150** [comment]
uploading also works #8

**Comment by mikk150** [comment]
> So ready to merge from your side?

so yes, ready to merge

File: composer.json [hunk 1]
```diff
@@ -33,7 +33,6 @@
         "league/flysystem-aws-s3-v3": "~1.0",
         "league/flysystem-azure": "~1.0",
         "league/flysystem-cached-adapter": "~1.0",
-        "league/flysystem-copy": "~1.0",
```
Comment (schmunk42) [inline composer.json:36]: Why did you remove this?

Comment (mikk150) [reply composer.json:36]: https://github.com/thephpleague/flysystem-copy/issues/16

TLDR - This service does not exist since 2016

**Comment by schmunk42** [comment]
Thank you!
